azure/template.json

```diff
@@ -5,6 +5,10 @@
         "resourceEnvironmentName": {
             "type": "string"
         },
+        "version": {
```

Comment:
I think this should just be hardcoded to 1.0 in the app settings to reduce the amount of parameters. Fine as it is though